google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/CloudSdkPanel.java

```diff
@@ -148,7 +148,6 @@ protected void showWarning(final String message) {
       public void run() {
         warningMessage.setText(message);
         warningMessage.setVisible(true);
-        warningMessage.updateUI();
```

Comment:
Everywhere I tested CloudSdkPanel, the message gets properly updated without these calls.

In the case you described, the message just wraps (which I actually thought impossible in UI forms :o), which I think is the correct behaviour?

Are there any other cases I should be aware of?